step.yml

```diff
@@ -0,0 +1,175 @@
+      description: |-
+        Set it as it is shown in
+        Xcode's device selection dropdown UI.
+
+        A couple of format examples (the
+        actual available options depend on which versions
+        are installed):
+
+        * "8.4"
+        * latest
+      is_required: "true"
+  - simulator_platform: iOS
+    opts:
+      title: Platform
+      description: |-
+        Set it as it is shown in Xcode's device selection dropdown UI.
+
+        A couple of examples (the
+        actual available options depend on which versions
+        are installed):
+
+        * iOS Simulator
+        * tvOS Simulator
+      value_options:
+      - iOS
+      - tvOS
+      is_required: "true"
+  - configuration: Debug
+    opts:
+      title: "Configuration name"
+      description: |-
+        (optional) The configuration to use. By default your Scheme
+        defines which configuration (Debug, Release, ...) should be used,
+        but you can overwrite it with this option.
+
+        **Make sure that the Configuration you specify actually exists
+        in your Xcode Project**. If it does not, if you have a typo
+        in the value of this input Xcode will simply use the Configuration
+        specified by the Scheme and will silently ignore this parameter!
+      is_required: "true"
+  - xcodebuild_options: ""
+    opts:
+      category: Debug
+      title: Additional options for xcodebuild call
+      description: |-
+        Options added to the end of the xcodebuild call.
+
+        You can use multiple options, separated by a space
+        character. Example: `-xcconfig PATH -verbose`
+  - workdir: $BITRISE_SOURCE_DIR
+    opts:
+      category: Debug
+      title: Working directory
+      summary: ""
+      description: |-
+        Working directory of the step.
+        You can leave it empty to don't change it.
+  - output_dir: $BITRISE_DEPLOY_DIR
+    opts:
+      category: Debug
+      title: Output directory path
+      summary: ""
+      description: |-
+        This directory will contain the generated .app artifacts.
+      is_required: "true"
+  - is_clean_build: "no"
+    opts:
+      category: Debug
+      title: Do a clean Xcode build before the build?
+      value_options:
+        - "yes"
+        - "no"
+      is_required: "true"
+  - output_tool: xcpretty
+    opts:
+      category: Debug
+      title: Output tool
+      description: |-
+        If output_tool is set to xcpretty, the xcodebuild output will be prettified by xcpretty.
+        If output_tool is set to xcodebuild, the raw xcodebuild output will be printed.
+      value_options:
+      - xcpretty
+      - xcodebuild
+      is_required: "true"
+      is_expand: "false"
+  - verbose_log: "no"
+    opts:
+      category: Debug
+      title: Enable verbose logging?
+      description: Enable verbose logging?
+      is_required: "true"
+      value_options:
+      - "yes"
+      - "no"
+
+outputs:
+  - BITRISE_EXPORTED_SIMULATOR_BUILDS_PATH_MAP:
```

Comment:
`BITRISE_APP_DIR_PATH` like in the xcode-archive step: https://github.com/bitrise-io/steps-xcode-archive/blob/master/step.yml#L292
This output should store the __main__ target .app's path

and `BITRISE_APP_DIR_PATH_LIST` like in the android-build step: https://github.com/bitrise-steplib/bitrise-step-android-build/blob/master/step.yml#L86

This output should store the __main__ target .app's path + every dependent target's .app path